services/api/routes/authenticated.js

```diff
@@ -190,7 +192,8 @@ var routes = [
                     Joi.number().integer().required(),
                     Joi.string().regex(/^\$(\d+)\.(.*)$/)
                   ),
-                  title: Joi.string().max(256).required()
+                  title: Joi.string().max(256).required(),
```

Comment:
Title was long because it served as hybrid title/description. 140 is fine for description. If no titles are currently longer than 140, we can reduce that length to match. Either way, I wouldn't worry about this too much.